From b83d78fc23a42ae844062ae3a953ae7426f2cf58 Mon Sep 17 00:00:00 2001
From: Andy CrossGate Yan <GeForce8800Ultra@gmail.com>
Date: Sat, 24 Sep 2022 11:40:52 +0000
Subject: [PATCH 1/2] vendor_lineage: Commonly build some packages

These were in handheld_product.mk and thus weren't included in GSI targets:
Contacts, DeskClock, Gallery2, SettingsIntelligence, fwb overlays

Change-Id: I00cc5918164ed70c612c5caa31eb8d2d0cd05b62
---
 config/common_full.mk   | 3 +++
 config/common_mobile.mk | 4 +++-
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/config/common_full.mk b/config/common_full.mk
index 029ea068..a30b2523 100644
--- a/config/common_full.mk
+++ b/config/common_full.mk
@@ -10,8 +10,11 @@ $(call inherit-product-if-exists, external/google-fonts/rubik/fonts.mk)
 # Apps
 PRODUCT_PACKAGES += \
     Aperture \
+    Contacts \
+    DeskClock \
     Eleven \
     Etar \
+    Gallery2 \
     Profiles \
     Recorder \
     Seedvault
diff --git a/config/common_mobile.mk b/config/common_mobile.mk
index fc5a864b..e2f2ada2 100644
--- a/config/common_mobile.mk
+++ b/config/common_mobile.mk
@@ -16,7 +16,9 @@ PRODUCT_PRODUCT_PROPERTIES += \
 PRODUCT_PACKAGES += \
     Backgrounds \
     ExactCalculator \
-    Jelly
+    Jelly \
+    SettingsIntelligence \
+    frameworks-base-overlays
 
 ifeq ($(PRODUCT_TYPE), go)
 PRODUCT_PACKAGES += \
-- 
2.34.1

